src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/ZipNameNotMatchingMetaDataException.java

```diff
@@ -1,6 +1,6 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
 
-public class ZipNameNotMatchingMetaDataException extends InvalidMetafileException {
+public class ZipNameNotMatchingMetaDataException extends RejectionException implements InvalidMetafileException {
```

Comment:
All exceptions that previously extended `InvalidMetadatafileException` should now extend `InvalidEnvelopeException`. It, in turn, along with `PaymentsDisabledException` and `ServiceDisabledException`, should extend `RejectionException`.

Reply:
so InvalidEnvelopeException will be interface?

Reply:
On a second thought, maybe we could get rid of `InvalidEnvelopeException` and just replace it with `RejectionException`. Technically, it doesn't add any benefit.

Reply:
> so InvalidEnvelopeException will be interface?

I think it should be completely replaced by `RejectionException`.